spec/models/user_spec.rb

```diff
@@ -1,5 +1,24 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe 'validates' do
+    subject { User.new name: 'John Doe' }
+
+    before { subject.save }
+
+    it 'should not be blank' do
+      subject.name = ''
+      expect(subject).to_not be_valid
+    end
+
+    it 'should have a name' do
+      subject.name = nil
+      expect(subject).to_not be_valid
+    end
+
```

Comment:
- [ ] You should test your models' methods as requested in the project requirements. Please add tests for `my_recent_posts` method.

> Add unit specs for all of your models' **methods** and validations.